dts/arm/renesas/ra/ra8/ra8x1.dtsi

```diff
@@ -557,6 +557,36 @@
 				status = "disabled";
 			};
 		};
+
+		sdhc0: sdhc@40252000 {
+			compatible = "renesas,ra-sdhc";
+			channel = <0>;
+			bus-width = <4>;
+			block-size = <512>;
```

Comment:
I'm not clear what the `block-size` property is here for- is this a property of the hardware itself? IE can the hardware only support cards of this block size?

Reply:
I use `block-size = <512>;` as a default value when init the driver in `R_SDHI_Open`,  it belongs to configs of struct `st_sdmmc_cfg`. In source, I get it by `DT_INST_PROP`, and user can modify it by overlay this device tree by themself

Reply:
See my comment here: https://github.com/zephyrproject-rtos/zephyr/pull/79862/files#r1854354437. I do not think we should be setting this property on the hardware- it is a property of each transfer request, not a fixed value.